libs/core/src/lib/breadcrumb/breadcrumb.component.ts

```diff
@@ -123,7 +123,7 @@ export class BreadcrumbComponent implements OnInit, AfterViewInit {
      * if original element had router link we are proxying click to that element
      * */
     itemClicked(breadcrumbItem: any, $event: any): void {
```

Comment:
Please, add the type too, so it does not happen again

Reply:
Done 👍